eduiddashboard/views/nins.py

```diff
@@ -195,7 +195,7 @@ def send_letter(request, user, nin):
                 'Please try again later.')
     if response.status_code == 200:
         expires = response.json()['letter_expires']
-        expires = datetime.utcfromtimestamp(expires)
+        expires = datetime.utcfromtimestamp(float(expires))
```

Comment:
Yes you are right. I was testing with turq and it was giving me strings, and when you feed strings to datetime.fromtimestamp it complains with " TypeError: a float is required". But that's not exactly true, an int also makes it happy. I'll change it.